<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="72182" started-at="2021-05-31T06:20:38Z" finished-at="2021-05-31T06:21:50Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="37557" started-at="2021-05-31T06:20:38Z" finished-at="2021-05-31T06:21:16Z">
      <class name="academy.HomePage">
        <test-method status="PASS" signature="initialize()[pri:0, instance:academy.HomePage@176b75f7]" name="initialize" is-config="true" duration-ms="17699" started-at="2021-05-31T06:20:38Z" finished-at="2021-05-31T06:20:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="basePageNavigation(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:academy.HomePage@176b75f7]" name="basePageNavigation" duration-ms="19702" started-at="2021-05-31T06:20:56Z" data-provider="data" finished-at="2021-05-31T06:21:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[karthik.sachin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[karthi1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Restricted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
      </class> <!-- academy.HomePage -->
    </test> <!-- Test -->
    <test name="Test1" duration-ms="31334" started-at="2021-05-31T06:21:19Z" finished-at="2021-05-31T06:21:50Z">
      <class name="academy.ValidateTitle">
        <test-method status="PASS" signature="initialize()[pri:0, instance:academy.ValidateTitle@20f5281c]" name="initialize" is-config="true" duration-ms="21385" started-at="2021-05-31T06:21:19Z" finished-at="2021-05-31T06:21:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="basePageNavigation()[pri:0, instance:academy.ValidateTitle@20f5281c]" name="basePageNavigation" duration-ms="1556" started-at="2021-05-31T06:21:40Z" finished-at="2021-05-31T06:21:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATUREd COURSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATUREd COURSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at academy.ValidateTitle.basePageNavigation(ValidateTitle.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="Breakdown()[pri:0, instance:academy.ValidateTitle@20f5281c]" name="Breakdown" is-config="true" duration-ms="3318" started-at="2021-05-31T06:21:47Z" finished-at="2021-05-31T06:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Breakdown -->
      </class> <!-- academy.ValidateTitle -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
